design-proposals/edge-infra-cve-tracking.md

```diff
@@ -0,0 +1,184 @@
+# Design Proposal: CVE Tracking for OS Profile and Edge Nodes
```

Comment:
@krishnajs has a good point that we shoudl define what CVE info we expect and want to show 
We should add a section and include the following elements that I think are worth showing for a CVE:
CVE ID
Link to CVE description/impact 
Priority 
Affected packages

Reply:
Done, please take a look.